src/modules/CRUD/components/Nav/NavIcon/NavIcon.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from '@blueprintjs/core';
+
+const NavIcon = ({ src, alt = '', isGroup, ...rest }) =>
+  (src
+    ? (
+      <span {...rest} className="bp3-icon">
```

Comment:
You can manage className coming from props by desctructuring it from `props`, then applying with your addition:

```suggestion
      <span {...rest} className={`${className} bp3-icon`}>
```